INSTALL.md

```diff
@@ -49,198 +31,135 @@ To update an existing clone you can use the following commands:
 You can use [Git for Windows](https://git-for-windows.github.io/) to
 clone the repository on Windows.
 
-# Dependencies
+## Backends
+
+LibreSprite can be compiled with two different backends:
+
+1. Allegro backend (Windows, Linux)
+2. Skia backend (Windows, Mac OS X)
 
-To compile LibeSprite you will need:
+## Dependencies
 
-* The latest version of [CMake](http://www.cmake.org/) (3.4 or greater)
-* [Ninja](https://ninja-build.org) build system
-* (Windows and MacOS only) [vcpkg](https://vcpkg.io/en/getting-started.html)
-  package manager
+You'll need the following dependencies to compile LibreSprite:
 
-LibreSprite can be compiled with two different back-ends:
+* freetype (FTL): https://www.freetype.org
+* giflib (MIT): http://giflib.sourceforge.net
+* gtest (BSD-3-Clause): https://github.com/google/googletest
+* libjpeg-turbo (BSD-3-Clause IJG Zlib): https://www.libjpeg-turbo.org
+* libpng (Libpng): http://www.libpng.org
+* libwebp (BSD-3-Clause) - https://developers.google.com/speed/webp
+* libx11 (XFree86): https://xorg.freedesktop.org
+* libxcursor (MIT): https://xorg.freedesktop.org
+* pixman (MIT): http://www.pixman.org
+* tinyxml (Zlib): http://www.grinninglizard.com/tinyxml
+* zlib (Zlib): https://www.zlib.net
```

Comment:
This is pretty useless, they're listed both in Linux and Windows dependencies section

Reply:
I think it's still a good idea to link to the project homepages and specify their licenses. It will also make it easier to figure out which packages need to be installed on other linux distributions.

Reply:
Projects are already linked in here https://github.com/LibreSprite/LibreSprite/blob/master/CONTRIBUTORS.md
Also, LibreSprite is a GPL project, so you kind of expect that it only runs free software libraries (permissive or not). Lastly, I think a Fedora/Arch user knows how to understand which libraries are their package manager equivalent by looking at Debian/Ubuntu ones

Reply:
I was thinking we might want to get rid of CONTRIBUTORS.md and simply link to the [contributors page](https://github.com/LibreSprite/LibreSprite/contributors). If we did then we would want to keep that information here.

Reply:
I disagree: the contributors page is something you can only see on GitHub. The file, on the contrary, it doesn't belong to any specific platform. In the future there could be some prerequisites to be added into that file, but it's out of the PR scope